src/index/updater.rs

```diff
@@ -626,7 +633,12 @@ impl<'index> Updater<'_> {
     index_inscriptions: bool,
   ) -> Result {
     if index_inscriptions {
-      inscription_updater.index_envelopes(tx, txid, Some(input_sat_ranges))?;
+      inscription_updater.index_envelopes(
```

Comment:
Can the inscription updater be constructed with the event sender, so we don't have to pass it through?